submissions/BlueLamantine/index.html

```diff
@@ -0,0 +1,677 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no"
+    />
+    <meta
+      name="description"
+      content="Find the newest movies near you, watch movie trailers and buy movie tickets."
+    />
+    <title>Cinema</title>
+    <link rel="shortcut icon" href="assets/img/favicon.ico" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+  <body>
+    <header class="movie-block">
+      <nav class="nav">
+        <ul class="nav__list" aria-label="Back to home page">
+          <li class="nav__item">
+            <a href="#" class="nav__link icon">Back</a>
+          </li>
+        </ul>
+        <ol class="breadcrumbs" aria-label="Breadcrumbs">
+          <li class="breadcrumbs__item icon">
+            <a
+              href="#"
+              class="breadcrumbs__link breadcrumbs__active"
+              aria-current="page"
+              >Seating of choice</a
+            >
+          </li>
+          <li class="breadcrumbs__item icon">
+            <a href="#" class="breadcrumbs__link" tabindex="-1">Payment</a>
+          </li>
+          <li class="breadcrumbs__item icon">
+            <a href="#" class="breadcrumbs__link" tabindex="-1">Finish</a>
+          </li>
+        </ol>
+      </nav>
+      <div class="movie">
+        <img
+          src="./assets/img/interstellar_poster.jpg"
+          class="poster"
+          alt="Movie poster"
+        />
+        <section class="movie__description">
+          <h1 class="movie__header" title="Movie title" tabindex="0">
+            Interstellar
+          </h1>
+          <p class="movie__info" title="Genre of Movie" tabindex="0">
+            Adventure, Drama, Sci-Fi
+          </p>
+          <p class="movie__info">
+            <time datetime="PT2H49M" tabindex="0">2h 49m</time>
+            <span class="pg" title="Movie rating" tabindex="0">12+</span>
+          </p>
+
+          <a
+            href="https://www.imdb.com/video/vi1586278169?playlistId=tt0816692&ref_=tt_ov_vi"
+            class="trailer__link icon"
+            >Watch trailer</a
+          >
+        </section>
+      </div>
+    </header>
+
+    <main>
+      <form id="ticketsForm" class="booking">
+        <section class="calendar">
+          <fieldset class="booking__date">
+            <legend class="booking__headline">Date</legend>
+
+            <div class="slider__date">
+              <div
```

Comment:
Why you don't use `button` for this element?

Reply:
A lot of slider-libraries use div for creating slider elements. But OK, now it is a ```button``` with type=button (cause I want to use other buttons in a form without it submitting)